src/main/java/seedu/duke/data/exception/SystemException.java

```diff
@@ -35,15 +35,21 @@
         EXCEPTION_MISSING_INDEX_PREFIX("Missing index prefix!"),
         EXCEPTION_MISSING_INDEX("Missing index!"),
 
+        EXCEPTION_MISSING_LINE("Missing line content!"),
```

Comment:
Not 100% sure if a user will understand what exactly Missing line content is. maybe can phrase it better?

Reply:
it just means that the contents for that line is missing